index.js

```diff
@@ -4,16 +4,23 @@ const { WebClient } = require('@slack/web-api');
 const flatten = require('flat');
 const axios = require('axios');
 
+const SLACK_WEBHOOK_TYPES = {
+    WORKFLOW_BUILDER: 'BUILDER',
+    WORKFLOW_INCOMING: 'INCOMING'
```

Comment:
As the "Incoming Webhooks" feature is not part of workflows / workflow builder, `WORKFLOW_INCOMING` does not make sense. Also, the value "BUILDER" can be confusing. How about renaming this way? Thoughts? @stevengill 
```suggestion
    WORKFLOW_TRIGGER: 'WORKFLOW_TRIGGER',
    INCOMING_WEBHOOK: 'INCOMING_WEBHOOK'
```

Reply:
I agree @seratch